docs/ENV.md

```diff
@@ -20,7 +20,7 @@ Arguments:
  - `%1` = Emote set ID
 
 ### CHATTERINO2_IMAGE_UPLOADER_URL
-Used to change the URL that Chatterino2 uses when trying to paste an image into chat. This can be used for hosting the uploaded images yourself.
+Used to change the URL that Chatterino2 uses when trying to paste an image into chat. This can be used for hosting the uploaded images yourself.  
```

Comment:
Minor nitpick: you seem to have left some spaces at the end of this line.

Reply:
@Mm2PL I added those on purpose, so there's a visible newline between description and Default value